### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
-# nva-institution-api
-API for administration of Institutions that are Customers in NVA
+# NVA Customer API
+API for administration of Institutions that are Customers in NVA.
+
+## Deploy considerations
```
Comment (brinxmat) [inline README.md:4]: Deployment

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+
+Read this before deploying this template.
+
+### Resources outside SAM template
+
+These resources are defined outside the SAM template but used in the template.
```
Comment (brinxmat) [inline README.md:10]: , but

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (norrs) [reply README.md:10]: A link to nva-deploy doesn't hurt.

**Review by norrs — CHANGES_REQUESTED** [review]
The environment variables defined in Environment should be documented as they act as configuration for the application. Application configuration is nice to document.

**Review by brinxmat — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,25 @@
...
+### Resources outside SAM template
+
+These resources are defined outside the SAM template, but used in the template. See details below.
+
+#### Customers DynamoDB Table
+
+Defined by template in 
+[cloudformation\dynamodb_customer_table.yml](cloudformation\dynamodb_customer_table.yml) and passed 
+to SAM template as parameters with table name and index names. Ensure table and indices is up to 
+date before deploying new SAM template.
```
Comment (norrs) [inline README.md:21]: This is good enough for now, but ideally we want these dynamodb changes to be defined in `template.yaml` as for the rest of the service when we know how to handle our migrations on persistent databases with cloudformation.

**Review by norrs — APPROVED** [review]

**Pull request merged**